src/resources/Microsoft.Authorization/policySetDefinitions/Deploy-MDFC-Config.json

```
@@ -8,7 +8,7 @@
         "displayName": "Deploy Microsoft Defender for Cloud configuration",
         "description": "Deploy Microsoft Defender for Cloud configuration",
         "metadata": {
-            "version": "5.0.1",
+            "version": "5.1.0",
```

```suggestion
            "version": "6.0.0.",
```
Think this is a major as technically removing and adding a new policy to the initiative as per: https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Policies#versioning